Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml

```
@@ -0,0 +1,762 @@
+commonfields:
+  id: Gurucul-GRA
+  version: -1
+name: Gurucul-GRA
+display: Gurucul-GRA (Community Contribution)
+category: Analytics & SIEM
```

I believe the type of the pack/integration should be `Data Enrichment & Threat Intelligence`.
See: https://xsoar.pan.dev/docs/concepts/use-cases#data-enrichment--threat-intelligence
```suggestion
category: Data Enrichment & Threat Intelligence
```